website/docs/docs/cloud/configure-cloud-cli.md

```diff
@@ -95,6 +95,7 @@ To set environment variables in the dbt Cloud CLI for your dbt project:
 ## Use the dbt Cloud CLI
 
 - The dbt Cloud CLI uses the same set of [dbt commands](/reference/dbt-commands) and [MetricFlow commands](/docs/build/metricflow-commands) as dbt Core to execute the commands you provide. For example, use the [`dbt environment`](/reference/commands/dbt-environment) command to view your dbt Cloud configuration details.
+- You can run multiple different invocations or commands in parallel. For example, `dbt build` and `dbt parse`.  Note, that you're unable to run the same dbt commands in parallel. For example, running `dbt build` at the same time isn't supported.
```

Comment:
I hope to clarify a little to make sure the words here is easy for customer to understand.

Today we support run 2 invocations but it might be more in future, depends on user feedback.